### ISSUE TITLE
Might be missing something - how to delete old backups?

### ISSUE THREAD

**Issue by dohlin** [pbek/ownbackup#12]
Long story short, I also encountered the file signature failure issue noted in another issue after my upgrade to Nextcloud 11.0.2 - I was able to resolve it myself by deleting the app folder and re-downloading, which I see another user has also alluded to in that other thread.

My question is where/how these backups are stored - when I go back into admin settings, it still lists my original backup(s) there, despite the app essentially being a "new" install (due to the aforementioned issue).  I don't see any option to delete these old backup(s) and am not sure how I can go about doing that.  I even tried re-disabling, then "uninstalling" the app and re-installing once more, still old backup(s) are shown. 
 Any pointers?  Thanks!

**Comment by pbek** [comment]
Please take a look at the folder `data/ownbackup`.

**Comment by pbek** [comment]
The folders there are named by the timestamp they were created.

**Comment by dohlin** [comment]
Awesome, thanks for the prompt reply!  That would explain it I didn't even think to look there as I have my data folder outside my NC directory.  I assume I can simply delete any folders I want and that won't cause any issues with old pointers or anything?

Also, what kind of timestamp are these folders using?  In my case I only have one prior backup, but inside my data folder, and inside the 'ownbackup' folder I have 1 folder named '1488164460'.  Since I only have 1 prior backup it makes sense that this folder must be the one, but I could see that being very confusing if I had many prior backups and was searching for one in particular.  I even tried right-clicking > properties but didn't see anything there that offered any further clues.

Sorry if these questions are super basic as I'm still an Ubuntu/Nextcloud/etc. novice, but I'm always trying to learn.  Thanks!

**Comment by dohlin** [comment]
EDIT: Found the solution myself...in Ubuntu (16.04) using the command "date -d @$TIMESTAMP" where '$TIMESTAMP' is the numerical string (in my case 1488164460) gave me the information I was looking for.  Thank you for all your help, I'll mark this 'issue' as closed!

And to answer my other question (in case anyone else wonders the same thing and finds this post in the future), yes I was able to simply delete the folder inside of 'owncloud' (the one with the long numerical timestamp) without issue.
